.github/workflows/lint.yml

```diff
@@ -17,13 +17,18 @@ jobs:
       # This step sets up the variable steps.golangci-lint-version.outputs.v
       # to contain the version of golangci-lint (e.g. v1.54.2).
       # The version is taken from go.mod.
+      - name: Install ZeroMQ
```

Comment:
Why would installing ZeroMQ be necessary for linting?

Reply:
We need to install it for compiling also, since goczmq is a Go wrapper around the native C library libzmq, which is the core library for ZeroMQ messaging.